conjur/login/login_controller.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+
+"""
+LoginController module
+
+This module is the controller that facilitates all login actions
+required to successfully configure the netrc
```

Comment:
As I told you, you should remove all reference to `netrc` as a concept.
Use `Credentials` instead.